streamer/src/nonblocking/quic.rs

```diff
@@ -905,21 +910,36 @@ impl ConnectionTable {
         }
     }
 
-    fn remove_connection(&mut self, key: ConnectionTableKey, port: u16) -> bool {
+    fn remove_connection(
+        &mut self,
+        key: ConnectionTableKey,
+        port: u16,
+        maybe_stable_id: Option<usize>,
```

Comment:
I think we can drop the Option here to clarify -- in production code, the stable_id is always expected it will clarify the logic. The option is necessary only for ease of fixing test code. But I think it is probably better just fix the test code anyway.